composer.json

```diff
@@ -91,6 +91,7 @@
         "matthiasnoback/symfony-config-test": "^4.2",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
         "nelmio/api-doc-bundle": "^2.13.5",
+        "phpstan/phpstan": "^0.12.84",
```

Comment:
We should add the phpstan extension installer and phpstan/phpstan-phpunit.

Reply:
With the current level, does it provide something ? Nothing fails without it, but I can add it, yes.

Reply:
It adds extra rules.

Reply:
Solved